lighthouse-core/audits/deprecations.js

```diff
@@ -56,16 +56,34 @@ class Deprecations extends Audit {
   static audit(artifacts) {
     const entries = artifacts.ConsoleMessages;
 
-    const deprecations = entries.filter(log => log.source === 'deprecation')
-    // TODO(M91): Temporary ignore until Chrome M91 became stable version.
-    // M91 doesn't throw deprecation on ::-webkit-details-marker.
-    .filter(log => !log.text.includes('::-webkit-details-marker'))
-    .map(log => {
-      return {
-        value: log.text,
-        source: Audit.makeSourceLocationFromConsoleMessage(log),
-      };
-    });
+    let deprecations;
+
+    if (artifacts.InspectorIssues.deprecations?.length) {
```

Comment:
```suggestion
    if (artifacts.InspectorIssues.deprecations.length) {
```
maybe something changed while writing? `deprecations` isn't optional now